python/tests/test_1_online.py

```diff
@@ -382,6 +382,22 @@ def test_webxdc_download_on_demand(acfactory, data, lp):
     assert msgs_changed_event.data2 == 0
 
 
+def test_enable_mvbox_move(acfactory, lp):
+    (ac1,) = acfactory.get_online_accounts(1)
+
+    lp.sec("ac2: start without mvbox thread")
+    ac2 = acfactory.new_online_configuring_account(mvbox_move=False)
+    acfactory.bring_accounts_online()
+
+    lp.sec("ac2: configuring mvbox")
+    ac2.set_config("configured_mvbox_folder", "DeltaChat")
```

Comment:
Looks like this needs to be done by default, but i don't understand how other tests enabling `mvbox_move` work now